cpython/Makefile

```diff
@@ -78,6 +82,20 @@ $(INSTALL)/lib/libffi.a :
 	mkdir -p $(INSTALL)/lib
 	cp $(FFIBUILD)/target/lib/libffi.a $(INSTALL)/lib/
 
+$(INSTALL)/lib/libhiwire.a :
+	rm -rf $(HIWIREBUILD)
+	mkdir $(HIWIREBUILD)
+	(\
+		cd $(HIWIREBUILD) \
+		&& git init \
+		&& git fetch --depth 1 $(HIWIREREPO) $(HIWIRE_COMMIT) \
+		&& git checkout FETCH_HEAD \
+		&& . $(PYODIDE_ROOT)/emsdk/emsdk/emsdk_env.sh \
+		&& CC=emcc EMSCRIPTEN_DEDUPLICATE=1 EXTERN_FAIL=1 make \
+	)
+	cp -r $(HIWIREBUILD)/dist/lib $(INSTALL)/
+	cp -r $(HIWIREBUILD)/dist/include $(INSTALL)/include/hiwire
```

Comment:
nitpick: It would be nice if libhiwire have a `make install` subcommand (No actions required for this PR).

Reply:
I didn't do that because I don't really know how to Makefile but it probably isn't very hard.